Meta/lint-ports.py

```diff
@@ -347,6 +347,12 @@ def run():
     from_table_set = set(from_table.keys())
     ports_set = set(ports.keys())
 
+    # Dictionaries are sorted by the order of addition, converting to a set loses this property
+    if list(from_table.keys()) != sorted(from_table.keys(), key=str.lower):
+        all_good = False
+        print('AvailablePorts.md is not in the correct order, please ensure that all ports are sorted alphabetically',
+              'by their name.')
```

Comment:
```suggestion
        print(
            'AvailablePorts.md is not in the correct order, please ensure that all ports '
            'are sorted alphabetically by their name.'
        )
```

Let's also be helpful and print out the expected order (i.e. sorted keys), computers do it differently than a human would.

Reply:
Just the entire list of ports?

Reply:
sure, all the names in expected order